tests/framework/intf/interface.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package framework
+package intf
```

Comment:
Don't understand where `intf` name comes from. When importing package, it should be clear what it contains by it's name.

Reply:
`intf` stands for `interface`. I vaguely recall that it's common used when I was coding with C++ more than 10 years ago. 

Also FYI. https://www.acronymfinder.com/Interface-(INTF).html

Reply:
Never seen it in Go though. Please let's not isolate new contributors that haven't used C++ to just save 5 characters in name. 
Balance in naming is definitely important, for example it doesn't make sense to use full name for local variable, however package name should be clear and informative over short and cryptic.

Reply:
Note that `interface` is a key word in golang, I intentionally used "intf" so as not to conflict with the keyword.

Reply:
If you don't like "intf", do you have any better proposal?

Reply:
Changed to "`interfaces`". 

Please let me know if you still have any comment.